### DISCUSSION TITLE
Smart collection based on "in the last"

### DISCUSSION THREAD

**Discussion by bigdork** [meisnate12/Plex-Meta-Manager#254] [Q&A] [answered]
I've read over the smart builders page (https://github.com/meisnate12/Plex-Meta-Manager/wiki/Smart-Builders) but unclear how to create a smart collection based on the age of the media, like "in the last" or "not in the last?" `originally_available` requires a specific date

For instance:
![Screen Shot 2021-05-18 at 3 20 17 PM](https://user-images.githubusercontent.com/58086116/118710900-e9f14480-b7ec-11eb-95c6-e9ec2082711e.png)

How to do something like this with PMM?

**Comment by meisnate12** [comment, marked as answer]
If you look at the [Smart Filter Modifiers](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Smart-Builders#filter-modifiers) for added and originally_available you can use no modifier to represent "in the last" (days only) or the .not modifier to represent "not in the last" (days only)

originally_available is the field that release date searches which is why it's called that

So basically this is what you're looking for 

originally_available: 90

**Reply by bigdork** [reply]
I'll give that a run, happy to see this functionality.
